### ISSUE TITLE
How to use this?

### ISSUE THREAD

**Issue by ayprof** [Sappharad/Lastpass-SSO-DotNET#1]
I'm trying to use the lastpass-cli but my organization's LastPass account is federated to AzureAD/EntraID and won't let me login.  I'm trying to get your fix up and running but I'm a bit confused on the steps.  I have the lastpass-cli built from the sso branch from jnewbigin and I can see where the login jumps to want the SSO fragment, but I can't figure out how to use your project for the next step.  Do you have any directions on how to get this working?

**Comment by Sappharad** [comment]
I don't remember if I ever got this working with the lastpass CLI, even the fork. I developed it specifically to use with lastpass-sharp, then discovered that library was already deprecated in favor of the password-manager-access library by the same author.

The patch for lastpass-sharp was attached to the thread you replied to earlier today in the password-manager-access issue.
[lastpasssharp_sso.txt](https://github.com/Sappharad/Lastpass-SSO-DotNET/files/14105922/lastpasssharp_sso.txt)
You would call the sample code here to get the password and fragment, and then pass it to lastpass-sharp to login with it. The sample app in 
https://github.com/Sappharad/Lastpass-SSO-DotNET/blob/main/lastpass-sso-net/Program.cs
is supposed to work in place of the cli-tool that the lastpass-cli okta fork uses, but I don't remember if that works or not. At the very least, running it by itself will give you the password and fragment values required to use that fork.

In short, answer to "How to use this" is that you don't use it alone, it's designed to be used with a lastpass tool or library that already supports SSO and just needs the password + fragment values from you. After I wrote it and got it working, I ended up not even using it because the application that I wanted to integrate lastpass support into couldn't run on the machine that I was writing the application for because it was a shared server.

It's here as a free example of how to do it if someone else ever needs it, but I no longer have the ability to test it after migrating away from LastPass.

Good luck.
